listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,96 @@
+@import "theme/theme.less";
+
+#recommendations {
+  &:extend(#listens);
+  padding: 10px;
+
+  > .recommendation-card {
+    padding: 15px;
+    padding-left: 4em;
+    margin-bottom: 7px;
+    max-height: 72px;
+    display: flex;
+
+    @media (max-width: @screen-sm-min) {
+      & {
+        padding-left: 30px;
+      }
+    }
+
+    &.current-recommendation {
+      background-color: rgba(217, 237, 247, 0.3) !important ;
+    }
+
+    .track-details {
+      flex: 1;
+      overflow: hidden;
+
+      > * {
+        overflow-x: hidden;
+        text-overflow: ellipsis;
+        white-space:nowrap;
+        margin-bottom: 0px;
+      }
+    }
+
+    a {
+      word-break: break-word;
+    }
+
+    .recommendation-controls {
+
+      flex: 0 140px;
+
+      &.open .dropdown-menu {
+        display: flex;
+        top: auto;
```

Comment:
Relating to my comment https://github.com/metabrainz/listenbrainz-server/pull/1197#issuecomment-757916336

You can simply add 
```less:
> * {
    flex: 1;
}
``` inside this block.